### PR TITLE
Create Avatar component to replace react-native-elements (1/2)

### LINKED ISSUE #210: Create Avatar component
## Describe the feature

Similar to react-native-elements's Avatar component, except it always implements `FastImage` and has different size enums.

### PR DESCRIPTION
**Pull request by idelmr**
Fixes #210

## Description

* Created Avatar component to work *almost* like react-native-elements's Avatar but with our defaults and the `FastImage` component

## Types of changes

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other 

## Checklist:

- [x] My code follows the code style of this project found on [Contribution guidelines](https://github.com/SHPEUCF/shpeucfapp/wiki/Contribution-guidelines).
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: src/components/general/Avatar.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import FastImage from "react-native-fast-image";
+import { TouchableOpacity, Text, Dimensions } from "react-native";
+
+const { height } = Dimensions.get("screen");
```
Comment (HanielDiaz) [inline src/components/general/Avatar.js:5]: disgusting

File: src/components/general/Avatar.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+			size = height * 0.05;
+			break;
+		case "medium":
+			size = height * 0.09;
+			break;
+		case "large":
+			size = height * 0.15;
+			break;
+		case "xlarge":
+			size = height * 0.3;
```
Comment (HanielDiaz) [inline src/components/general/Avatar.js:36]: one of these is not like the other....
talking about 0.09

File: src/components/general/Avatar.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+	return (
+		<TouchableOpacity
+			activeOpacity = { (!onPress) ? 1 : 0.2 }
+			onPress = { () => onPress && onPress() }
+			style = { [style, center, { height: size, width: size, borderRadius }] }
+		>
+			{ source
+				? <FastImage style = {{ height: size, width: size, borderRadius }} source = { source } />
+				: <Text style = { [titleStyle, { fontSize: size / 2, color: "white" } ] }>{ title }</Text>
+			}
```
Comment (HanielDiaz) [inline src/components/general/Avatar.js:58]: Not gonna lie... i don't know what this does. Would be nice if this was in a function that explained what it did/ why it's necessary

**Review by HanielDiaz — DISMISSED** [review]
yeah idk it do be like it do sometimes

**Review by csepulveda7 — APPROVED** [review]
:)

**Pull request merged**